sklearn/svm/base.py

```diff
@@ -779,6 +779,15 @@ def _fit_liblinear(X, y, C, fit_intercept, intercept_scaling, class_weight,
     if verbose:
         print('[LibLinear]', end='')
 
+    # LinearSVC breaks when intercept_scaling is <= 0
+    if fit_intercept is True:
+        if intercept_scaling is None:
```

Comment:
yes, but with the ``fit_intercept is True`` you don't need it any more. That was kind of the point.